examples/ContainerApp/ContainerApp.Tests.ps1

```diff
@@ -1,13 +1,13 @@
-BeforeAll {
+﻿BeforeAll {
   Import-Module Az.InfrastructureTesting
 
   $Script:rgName = 'rg-test'
-  $Script:conAppName = 'conAppBenchPressTest'
   $Script:location = 'westus3'
 }
 
 Describe 'Verify Container Application' {
   BeforeAll {
+    $Script:conAppName = 'conAppBenchPressTest'
```

Comment:
```suggestion
    $Script:containerAppName = 'containerAppBenchPressTest'
```
To be consistent with the no app variable name :)

Reply:
Done!